source/administration/config-settings.rst

```diff
@@ -2212,6 +2212,16 @@ Email notifications will be sent to this email address, and this email address m
 | This feature's ``config.json`` setting is ``"EmailAttribute": ""`` with string input.                                                                                |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Profile Picture Attribute
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The attribute in the AD/LDAP server used to synchronize (and lock) the profile picture used in Mattermost.
+
+The mattermost server will replace the user’s profile image upon login. Note, this only occurs when the user either does not have a profile image in mattermost, or the current profile image does not match the image associated with that user in AD/LDAP.
```

Comment:
Should mattermost be capitalized? 2 places. The statement after "Note:" seems confusing. "This ONLY occurs...", when else would it occur?
Should there be something about occurs at login, not with normal sync.